ifrs17/Import/Importers.ipynb

```diff
@@ -444,28 +467,19 @@
         {
             "cell_type": "code",
             "source": [
-                "public async Task<ImportArgs[]> GetTargetArgsAsync(ImportArgs mainArgs)",
+                "public async Task<ImportArgs[]> GettargetArgsListAsync(ImportArgs mainArgs)",
```

Comment:
Typo on Target. 
Why do you call a list? You don't return a List. I would keep the old name.

Reply:
renamed to GetAllArgsAsync